auth/server/base/src/main/java/org/wildfly/security/auth/server/MechanismInformation.java

```diff
@@ -53,6 +55,10 @@ public interface MechanismInformation {
      */
     String getProtocol();
 
+    default Map<String, ?> getProperties(){
```

Comment:
Is this needed? There are some other places below where I see the same method added but I might be missing where it is used.